index.bs

```diff
@@ -2789,6 +2789,50 @@ the [=authenticator=] MUST:
     attStmtTemplate .within $$attStmtType
     ```
 
+### Signature Formats ### {#signature-attestation-types}
+    - For COSEAlgorithmIdentifier -7 (ES256), `sig` contains a pair of 32-byte integers,
+        R followed by S, generated with ECDSA using Curve P-256 and SHA256 encoded as
+        ASN.1 DER format defined in [[ANSI.X9-62.2005]] and [[FIPS.186-4.2013]].
+
+        Note: As CTAP1/U2F devices are already producing signatures in this format, CTAP2
+        devices will also produce signatures in same format for consistency. Otherwise,
```

Comment:
Change "in same format for consistency" to "in same format, for consistency reasons"

Reply:
Changed.